### PR TITLE
[improvement](filecache)Change the hash field of the backend

### PR DESCRIPTION
**Pull request by zddr**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.
ip of backend may change
use id as a hash field

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.35 seconds
 stream load tsv:          480 seconds loaded 74807831229 Bytes, about 148 MB/s
 stream load json:         39 seconds loaded 2358488459 Bytes, about 57 MB/s
 stream load orc:          74 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230307073930_clickbench_pr_109924.html

**Comment by zddr** [comment]
run feut

**Comment by zddr** [comment]
run feut

**Comment by zddr** [comment]
run p0

File: fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java [hunk 1]
```diff
@@ -3251,7 +3250,7 @@ public void addBucketSeq(int bucketSeq) {
         }
 
         public FInstanceExecParam(TUniqueId id, TNetworkAddress host,
-                                  int perFragmentInstanceIdx, FragmentExecParams fragmentExecParams) {
+                int perFragmentInstanceIdx, FragmentExecParams fragmentExecParams) {
```
Comment (AshinGau) [inline fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java:3253]: Why change the indent format?

**Comment by zddr** [comment]
run buildall

**Comment by AshinGau** [comment]
LGTM

**Comment by zddr** [comment]
run p0

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**